src/lib/iframe-tools/identity-manager.js

```diff
@@ -87,12 +87,27 @@ export class IdentityManager extends IframeManager {
       return;
     }
 
-    debugLogger.log('[identity-manager:fetchIdentity] Fetch country request.');
-    this.fetchCountry().then((country) => {
+    const localStorageKey = 'pp-sdk-shouldReallyCheckCountry';
+    const LS = window.localStorage;
```

Comment:
Please wrap this window.localStorage usage in a try/catch. The `getItem()` and `setItem()` can throw errors if local storage is full or not available.

The belter utility library does have a `isLocalStorageEnabled()` function you can use which safely checks for localStorage support with a try/catch: https://github.com/krakenjs/belter/blob/2f03832059dabc709d296a624e9798a2e8d66f44/src/dom.js#L306